query-model/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/expression/impl/JpqlCurrentDate.kt

```diff
@@ -0,0 +1,6 @@
+package com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import java.sql.Date
+
+object JpqlCurrentDate : Expression<Date>
```

Comment:
Please add the `Internal annotation` because this class belongs to an implementation. 

```kotlin
@Internal
object JpqlCurrentDate : Expression<Date>
```